spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -582,7 +602,10 @@
 
           expect(job_analytics).to have_logged_event(
             'GetUspsProofingResultsJob: Enrollment status updated',
-            hash_including(reason: 'Enrollment has expired'),
+            hash_including(
+              minutes_since_established: (3.days.in_minutes..(3.days.in_minutes + 5)),
```

Comment:
this `(3.days.in_minutes..(3.days.in_minutes + 5))` is repeated a lot, maybe we could add a helper, or even a `let` value for it? like

```
def range_approximating(duration)
  duration..(duration + 5)
end

range_approximating(3.days.in_minutes)
```

Reply:
Helper added.